OSDOCS-6269: Updates intro. listing all resources and  listing nbdb/sbdb with OVN-K  i/c (#62156)
openshift/openshift-docs: modules/ovn-kubernetes-architecture-con.adoc

* **Cloud Management System (CMS)** - A platform specific client for OVN that provides a CMS specific plugin for OVN integration. The plugin translates the cloud management system's concept of the logical network configuration, stored in the CMS configuration database in a  CMS-specific  format, into an intermediate representation understood by OVN.
* **OVN Northbound database (`nbdb`)** - Stores the logical network configuration passed by the CMS plugin.
* **OVN Southbound database (`sbdb`)** - Stores the physical and logical network configuration state for OpenVswitch (OVS) system on each node, including tables that bind them.

> * **ovn-northd** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-node` pods.

Reviewer: * Existing content, but would be good to have a noun after `nbdb` and `sbdb` (not sure what they are aside from "components").
* In another module, you have "OVN north daemon" in plain text. Could this match the other list items above like so? 

```suggestion
* **OVN north daemon (`ovn-northd`)** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-node` pods.
```

* **ovn-controller** - This is the OVN agent that interacts with OVS and hypervisors, for any information or update that is needed for `sbdb`. The `ovn-controller` reads logical flows from the `sbdb`, translates them into `OpenFlow` flows and sends them to the node’s OVS daemon. The container name is `ovn-controller` and it runs in the `ovnkube-node` pods.

> The OVN northd, northbound database and southbound database run on each node in the cluster and mostly contain and process information that is local to that node.

Reviewer: ```suggestion
The OVN north daemon, northbound database and southbound database run on each node in the cluster and mostly contain and process information that is local to that node.
```

The OVN northbound database has the logical network configuration passed down to it by the cloud management system (CMS).
The OVN northbound Database contains the current desired state of the network, presented as a collection of logical ports, logical switches, logical routers, and more.
The `ovn-northd` (`northd` container) connects to the OVN northbound database and the OVN southbound database.
It translates the logical network configuration in terms of conventional network concepts, taken from the OVN northbound Database, into logical data path flows in the OVN southbound database.

> The OVN southbound database has physical and logical representations of the network and binding tables that link them together. It contains the chassis information of the node and other constructs like remote transit switch ports that are required to to connect to the other nodes in the cluster. The OVN southbound database also contains all the logic flows, the logic flows are shared with the `ovn-controller` process that runs on each node and the `ovn-controller` turns those into `OpenFlow` rules to program `Open vSwitch`.

Reviewer: ```suggestion
The OVN southbound database has physical and logical representations of the network and binding tables that link them together. It contains the chassis information of the node and other constructs like remote transit switch ports that are required to to connect to the other nodes in the cluster. The OVN southbound database also contains all the logic flows. The logic flows are shared with the `ovn-controller` process that runs on each node and the `ovn-controller` process turns those into `OpenFlow` rules to program `Open vSwitch`.
```
Is there a noun that can describe what `Open vSwitch` is?

Author: It is a network switch as far as I understand. I _think_ adding that after `OvS` sounds awkward though. Let me check with SME and see if we can come up with something better.

> The Kubernetes control plane nodes each contain an `ovnkube-control-plane` pod which does the central IPAM allocation for each node in the cluster. At any given time a single `ovnkube-control-plane` pod is the leader.

Reviewer: Define IPAM on first use?

> The OVN southbound database has physical and logical representations of the network and binding tables that link them together. Every node in the cluster is represented in the southbound database, and you can see the ports that are connected to it.

Reviewer 2: this might be a bit confusing without mentioning what local and remote are.
local node = node on which SBDB is on
remote node = all other nodes in the cluster.
